CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Where to start?
+
+This document describes everything to do with contributing to this repository,
+including commit style, coding style, submitting issues, and making pull
+requests.
+
+If you want to contribute and don't know where to start, we are a bunch of
+incredibly friendly people who are very eager to help you out! A good place to
+go is to look at our
+[issues list](https://github.com/hackersatcambridge/hac-website/issues)
+for anything tagged with the
+[`good first issue`](https://github.com/hackersatcambridge/hac-website/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
+label.
+
+# Issues
+
+In order to make our lives as easy as possible when submitting an issue, please
+try and make everything as clear as possible. Label the issue appropriately
+whether it's a
+[`bug`](https://github.com/hackersatcambridge/hac-website/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
+or a possible
+[`enhancement`](https://github.com/hackersatcambridge/hac-website/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement),
+and ensure that if it is a bug, that as much information as possible is there to
+help us reproduce it.
+
+# Pull Requests
+
+If the pull request should resolve any issues, please list them in a comment!
+
```

Comment:
We should also mention that frontend changes should have screenshots

Reply:
Could you do that pls? I'm on phone now.

Reply:
Done